mobilizon_reshare/config/config.py

```diff
@@ -41,54 +33,58 @@ def current_version() -> str:
         return fp.read()
 
 
-def build_settings(
-    settings_file: Optional[str] = None, validators: Optional[list[Validator]] = None
-):
+def get_settings_files_paths():
+
+    dirs = AppDirs(appname="mobilizon-reshare", version=current_version())
+    bundled_settings_path = pkg_resources.resource_filename(
+        "mobilizon_reshare", "settings.toml"
+    )
+    bundled_secrets_path = pkg_resources.resource_filename(
+        "mobilizon_reshare", ".secrets.toml"
+    )
+    return [
```

Comment:
Did you figure out how does dynaconf merge different settings files? Otherwise I believe it would be safer to choose the first readable file from this list.

Reply:
I believe we should let users deal with dynaconf themselves. Overriding its behavior would make it harder to delegate to Dynaconf's own documentation. If you do weird things with the configuration and have unreliable merges, it shouldn't be our problem.

Reply:
Yes but we are overriding Dynaconf behavior in the first place by giving it a list of paths instead of a single path. If we want to follow a KISS approch we shoud just use the user config dir as a single value, without the list. We can always open an issue if we want to change back